README.md

```diff
@@ -5,4 +5,4 @@
 | Short Desc        | A bot that filters text, images, and video based on user-defined tags.  |
 | Members Committed | Kabir Kwatra, Ivar Rydstrom, Raymond Wong, Aditya Dhir                  |
 | Skills We Got     | TBD                                                                     |
-| Skills Needed     | Also TBD                                                                |
+| Skills Needed     | Also TBD                                                                |
```

Comment:
remove all changes in this file

Reply:
@KabirKwatra how do I do that

Reply:
I did it for you because it was kinda complicated